src/next/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,331 @@
+import {
+  type HeroGradientProps,
+  type HeroSplitProps,
+  type HeroFloatingProps,
+  type HeroCopyLedProps,
+  type HeroFloatingImageProps,
+} from "~/common/Hero"
+import Button from "~/next/components/Button"
+
+export type HeroProps =
+  | HeroGradientProps
+  | HeroSplitProps
+  | HeroFloatingProps
+  | HeroCopyLedProps
+  | HeroFloatingImageProps
+
+const HeroGradient = ({
+  alignment = "left",
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: HeroGradientProps) => {
+  return (
+    <section
+      className="bg-cover bg-center bg-no-repeat h-[32rem]"
+      style={{
+        backgroundImage: `url('${backgroundUrl}')`,
+      }}
+    >
+      <div
+        className={`h-full w-full ${
+          alignment === "left" ? "bg-gradient-to-r" : "bg-gradient-to-l"
+        } from-[rgba(0,0,0,85%)] md:from-[rgba(0,0,0,75%)] xl:from-[rgba(0,0,0,65%)]`}
+      >
+        <div
+          className={`text-white p-6 flex flex-row items-end h-full ${
+            alignment === "left" ? "justify-start" : "justify-end"
+          } ${alignment === "left" ? "text-start" : "text-end"}`}
+        >
+          <div className="flex flex-col w-3/5 xl:w-2/5 gap-6">
+            <h1 className="text-6xl font-medium">{title}</h1>
+            {subtitle && <p>{subtitle}</p>}
```

Comment:
nit: from design the subtitle font size should be ~24px or `text-2xl` (for all the variants i think)
it does still look different after changing it but I think there's something going on with the weight idk why our weight:400 is much thicker

Reply:
Adjusted in 8ee7e6894f8ec730fbe37e1b68663fca2b02db4c.